logging/log_storage/log-storage-overview.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="log-storage-overview"]
+= About log storage
+:context: log-storage-overview
+
+toc::[]
+
+You can use an internal Loki or Elasticsearch log store on your cluster for storing logs, or you can use a xref:../../logging/log_collection_forwarding/log-forwarding.adoc#logging-create-clf_log-forwarding[`ClusterLogForwarder` custom resource (CR)] to forward logs to an external store.
+
+[id="log-storage-overview-types"]
+== Log storage types
+
+Loki is a horizontally scalable, highly available, multi-tenant log aggregation system currently offered as an alternative to Elasticsearch as a log store for the {logging}.
+
+Elasticsearch indexes incoming log records completely during ingestion. Loki only indexes a few fixed labels during ingestion and defers more complex parsing until after the logs have been stored. This means Loki can collect logs more quickly.
+
+include::modules/cluster-logging-about-es-logstore.adoc[leveloffset=+2]
+
```

Comment:
@abrennan89 I feel its good to have a "About the Loki log store" section here.

Reply:
We can, but that would be a follow up for another Jira. This PR is just to restructure existing content, not to fill gaps for content that doesn't exist yet.